CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ chandler is in a pre-1.0 state. This means that its APIs and behavior are subjec
 
 ## [Unreleased][]
 
-* Your contribution here!
```

Comment:
Can you put the `Your contribution here!` bullet back in? I like to keep in there as a instruction for future contributors.